examples/counter/tests/mod.rs

```diff
@@ -84,22 +84,22 @@ fn inc() {
     let clear = div
         .first_child()
         .unwrap()
-        .dyn_into::<HtmlElement>()
+        .dyn_into::<web_sys::HtmlElement>()
```

Comment:
I'm not certain these added turbofish are needed, the following command seems to run with just the other changes you proposed:
wasm-pack test --node